applications/concept-catalogue/src/main/kotlin/no/begrepskatalog/conceptregistration/HarvestEndpoint.kt

```diff
@@ -10,21 +12,23 @@ import org.slf4j.LoggerFactory
 import org.springframework.beans.factory.annotation.Value
 import org.springframework.http.ResponseEntity
 import org.springframework.web.bind.annotation.CrossOrigin
+import org.springframework.web.bind.annotation.RequestParam
 import org.springframework.web.bind.annotation.RestController
 import java.io.StringWriter
 import javax.servlet.http.HttpServletRequest
+import javax.validation.Valid
 
 
 @RestController
 @CrossOrigin(value = "*")
-class HarvestEndpoint(val sqlStore: SqlStore) {
+class HarvestEndpoint(val sqlStore: SqlStore) : CollectionsApi {
 
     @Value("\${application.baseURL}")
     lateinit var baseURL: String
 
     private val logger = LoggerFactory.getLogger(HarvestEndpoint::class.java)
 
-    fun harvest(httpServletRequest: HttpServletRequest?): ResponseEntity<String> {
+    override fun getCollections(httpServletRequest: HttpServletRequest, @ApiParam(value = "Ansvarlig virksomhet") @Valid @RequestParam(value = "publisher", required = false) publisher: String): ResponseEntity<Any> {
```

Comment:
Må du ha med alle annotasjonene for variablene, @ApiParam, @Valid osv, her i implementasjonen?

Reply:
Nei, testet nå og det funker uten. Godt poeng!